activity_browser/app/bwutils/multilca.py

```diff
@@ -9,18 +9,37 @@
 from .commontasks import wrap_text
 from .metadata import AB_metadata
 
+
 class MLCA(object):
-    # todo: update description
-    # todo: add characterized inventories
     """Wrapper class for performing LCA calculations with many functional units and LCIA methods.
 
-    Needs to be passed a ``calculation_setup`` name.
+    Needs to be passed a brightway ``calculation_setup`` name.
+
+    This class does not subclass the `LCA` class, and performs all
+    calculations upon instantiation.
+
+    Initialization creates `self.lca_scores`, which is a NumPy array
+    of LCA scores, with rows of functional units and columns of LCIA methods.
+    Ordering is the same as in the `calculation_setup`.
 
-    This class does not subclass the `LCA` class, and performs all calculations upon instantiation.
+    Class adapted from bw2calc.multi_lca.MultiLCA to include also
+    CONTRIBUTION ANALYSIS.
 
-    Initialization creates `self.lca_scores`, which is a NumPy array of LCA scores, with rows of functional units and columns of LCIA methods. Ordering is the same as in the `calculation_setup`.
+    Parameters
+    ----------
+    cs_name : str
+        Name of the calculation setup
 
-    Class adapted from bw2calc.multi_lca.MultiLCA to include also CONTRIBUTION ANALYSIS.
+    Attributes
```

Comment:
this is still quite incomplete. Best introduce also the different vectors and matrices that are used for storing the LCA results (e.g. process contributions); and also the indexing dictionaries

Reply:
Have just added all of the attributes of the class. It kind of feels like the comments in the __init__ method are superfluous now.